release/ray_release/glue.py

```diff
@@ -235,6 +255,7 @@ def run_release_test(
             cluster_manager.start_cluster(timeout=cluster_timeout)
 
         result.cluster_url = cluster_manager.get_cluster_url()
+        result.session_id = cluster_manager.cluster_id
```

Comment:
Let's please not call this session ID. This is confusing - it's even called cluster_id in the cluster manager. Session ID is a legacy name that is not used anywhere anymore